lucille-core/src/test/java/com/kmwllc/lucille/stage/ApplyRegexTest.java

```diff
@@ -62,6 +64,110 @@ public void testCapturingGroup() throws Exception {
     assertEquals("123", doc.getString("output"));
   }
 
+  @Test
+  public void testIgnoreCase() throws Exception {
+    Stage stage = factory.get("ApplyRegexTest/ignoreCase.conf");
+
+    Document doc1 = Document.create("1");
+    doc1.setField("input", "Hi abcdef");
+
+    Document doc2 = Document.create("1");
```

Comment:
I know this test doesn't look at doc ID, but I'd suggest giving different docs different IDs (so doc2 would get "2")... same comment for other tests here.